crypto/evp/evp_enc.c

```diff
@@ -186,6 +189,7 @@ static int evp_cipher_init_internal(EVP_CIPHER_CTX *ctx,
         ERR_raise(ERR_LIB_EVP, EVP_R_INITIALIZATION_ERROR);
         return 0;
 #else
+        /* TODO: query for pipeline support when `is_pipeline` */
```

Comment:
No, it is not covered. Currently, EVP is implicitly fetching a provider that takes precedence for a given nid. If that provider happens to not support pipeline, the cipher init fails.   
The TODO is to use "property query strings" for fetching pipeline supported provider among multiple. (only for `EVP_CipherPipelineInit()`). Since, this is covered in "Future Ideas" section of design document, I can remove the comment.

Reply:
Yes, please remove it.